### DISCUSSION TITLE
Data treatment prior to reconstruction

### DISCUSSION THREAD

**Discussion by lo-caviu** [ORTools4MTEX/ORTools#13] [Q&A] [answered]
Hi, I have recently performed parent grain construction for lath martensite following both Example 1 and 7.  My dataset consists of austenite that is stabilized to room temperature (not retained) and (likely; I intend to confirm the presence by the reconstruction) lath martensite and potentially local ferrite (I would like to localize the regions that are ferritic). 

The dataset is such that "parent austenite"-like grains are constructed prior to reconstruction, as can be seen below: 
_IPF Z child grains before reconstruction:_ 
![image](https://user-images.githubusercontent.com/106658708/171364979-b9ed095e-2466-403e-a47d-24afa34c0c74.png)
_Child boundary misorientation map before reconstruction:_
![image](https://user-images.githubusercontent.com/106658708/171365141-649b865f-3b24-4697-a219-f81ffc1a4fce.png)

Hence, I am not quite sure how to handle the original dataset prior to reconstruction considering I am already using low threshold values for grain boundaries and inner boundaries (2.5 and 1 degree, respectively).

**Comment by frankNiessen** [comment, marked as answer]
Hi lo-caviu,

I can see from your second figure that the austenite is contained on the upper region of your sample, whereas the bcc phase (ferrite or martensite) is mostly in the lower region. 
If you form such large bcc grains with gradually changing orientation it does not indicate martensite formation, as you would find more sharp interfaces and a hierarchical structure. You might want to assume an orientation relationship like K-S (`KS = orientation.KurdjumovSachs(job.csParent,job.csChild);`) to start with and create a grain boundary plot highlighting the angular misfit using function [plotmap_gb_misfit](https://github.com/ORTools4MTEX/ORTools/blob/develop/README.md#plotmap_gb_misfit) as showcased in [example 1](https://github.com/ORTools4MTEX/ORTools/blob/develop/ORTools_example1.m). Here a low misfit with the OR would indicate that the grain boundary could be a martensitic boundary. In case you cannot observe this it is unlikely that you have martensite. 

I hope this is helpful to you, 

Best wishes
Frank
